mix.exs

```diff
@@ -31,20 +31,20 @@ defmodule Membrane.WAV.Plugin.Mixfile do
 
   defp deps do
     [
-      {:membrane_core, "~> 0.8.1"},
-      {:membrane_caps_audio_raw, "~> 0.5.0"},
-      {:membrane_file_plugin, "~> 0.8.0", optional: true},
+      {:membrane_core, "~> 0.9.0"},
```

Comment:
The update to the new core usually involved migration to auto-demands. This version should include that I think, although it won't be a trivial demand removal, a change in logic will be required. PM if you need more details